src/xpk/commands/cluster.py

```diff
@@ -317,6 +322,30 @@ def cluster_create(args) -> None:
 
   install_kjob(args)
 
+  # Update memory limit of Jobset Controller Manager
+  k8s_api_client = setup_k8s_env(args)
+  jobset_manifest = update_jobset_manifest(args)
+  if jobset_manifest is None:
+    xpk_print(
+        "Updated jobset manifest is empty, not updating the jobset controller."
+    )
+  xpk_print("Updating Jobset Controller Manager")
+  return_code = apply_kubectl_manifest(k8s_api_client, [jobset_manifest])
+  if return_code != 0:
+    xpk_print(f'Updating Jobset Controller Manager returned ERROR {return_code}')
+    return return_code
+
+  kueue_manifest = update_kueue_manifest(args)
+  if kueue_manifest is None:
+    xpk_print(
+        "Updated Kueue manifest is empty, not updating the Kueue controller."
+    )
+  xpk_print("Updating Kueue Controller Manager")
+  return_code = apply_kubectl_manifest(k8s_api_client, [kueue_manifest])
+  if return_code != 0:
+    xpk_print(f'Updating Kueue Controller Manager returned ERROR {return_code}')
+    return return_code
```

Comment:
Let's make this into 2 self contained functions, one for setting the jobset manager memory, and another for kueue controller manager as well.